osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/stub/DlmsConnectionStub.java

```diff
@@ -164,6 +164,12 @@ public boolean hasMethodBeenInvoked(final MethodClass methodClass) {
         .anyMatch(parameter -> parameter.getId() == methodClass.getMethodId());
   }
 
+  public long hasMethodBeenInvokedTimes(final MethodClass methodClass) {
```

Comment:
Suggestion for name: `getMethodInvocationCount`

Reply:
implemented as suggested